src/pydata_sphinx_theme/assets/styles/extensions/_sphinx_design.scss

```diff
@@ -304,3 +304,35 @@ details.sd-dropdown {
     }
   }
 }
+
+/*******************************************************************************
+* Buttons (which in Sphinx Design are actually links that look like buttons)
+* ref: https://sphinx-design.readthedocs.io/en/pydata-theme/badges_buttons.html#buttons
+*/
+html {
+  .sd-btn {
+    min-width: 2.25rem;
+    padding: 0.3125rem 0.75rem 0.4375rem; // 5px 12px 7px
+
+    @include link-style-hover; // override Sphinx Design
+    &:hover {
+      text-decoration-thickness: 1px;
+    }
+  }
+
+  @each $name in $sd-semantic-color-names {
+    .sd-btn-#{$name},
+    .sd-btn-outline-#{$name} {
```

Comment:
This was the selector (sd-btn-outline-name) needed to apply these styles to the buttons on the [layout](https://pydata-sphinx-theme--1589.org.readthedocs.build/en/1589/user_guide/layout.html) page.

Reply:
I checked the Sphinx Design source code, searched for "sd-btn-" and these were the only two variations I found.